src/webrequest.js

```diff
@@ -55,6 +55,10 @@ function onBeforeRequest(details){
     recordFrame(details.tabId, details.frameId, details.parentFrameId, details.url);
   }
 
+  if (type == "ping" && details.tabId < 0 ){
```

Comment:
Are there ever ping requests that happen on visible tabs? Is there a reason to make this check? 
Also it might be having a comment which points to an explanation of what a ping request is, since this may be non obvious.